guides/12_Client_Event_Ingestion.md

```diff
@@ -0,0 +1,45 @@
+# Event Ingestion
+
+We send the [Amazon Chime SDK meeting events](https://aws.github.io/amazon-chime-sdk-js/modules/meetingevents.html#meeting-events-and-attributes) to the Amazon Chime backend to analyze meeting health trends or identify common failures. This helps to improve your meeting experience.
+
+## Sensitive attributes
+
+The Amazon Chime SDK for JavaScript will not send below sensitive attributes to the Amazon Chime backend.
+|Attribute|Description
+|--|--
+|`externalMeetingId`|The Amazon Chime SDK external meeting ID.
+|`externalUserId`|The Amazon Chime SDK external user ID that can indicate an identity managed by your application.
+   
+## Opt out of Event Ingestion
+   
+To opt out of event ingestion, provide `NoOpEventReporter` to `DefaultMeetingSession` while creating the meeting session.
```

Comment:
Is it simpler to just have a flag to disable event reporter?

Reply:
I had it first but then we thought it would be confusing to say disable it because then we have to clarify if builders can enable it or not and based on what factors and so on handle the logic around it. I synced with mobile and planned to go ahead with this one approach to keep it consistent within both platforms.